fb_kfukuhar.md

```
@@ -0,0 +1,999 @@
+# agenda:
+## process minishell
+### cmd excute
+#### relative path
+```bash
+minishell$ ls
+==2102562== Memcheck, a memory error detector
+==2102562== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
+==2102562== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
+==2102562== Command: /usr/bin/ls
+==2102562== 
+ Makefile	    get_next_line   ls_test.c   out1   out4	 test
```

Markdownlint (0.37.0)

Column: 10
Hard tabs

(MD010, no-hard-tabs)